http.go

```diff
@@ -17,6 +17,18 @@ import (
 	"github.com/valyala/bytebufferpool"
 )
 
+var (
+	requestBodyMaxLimit  = -1
+	responseBodyMaxLimit = -1
+)
+
+// SetBodySizePoolLimit set the max body limit in request and response
+// if the body size larger than the limit,it will be released
```

Comment:
```suggestion
// SetBodySizePoolLimit set the max body size for bodies to be returned to the pool.
// If the body size is larger it will be released instead of put back into the pool for reuse.
```